tests/test_core.py

```diff
@@ -1162,9 +1162,9 @@ def test_exceptions_uncaught_2(self):
           }
 
           if (std::uncaught_exception())
-            std::cout << "ERROR: uncaught_exception still set.";
+            std::cout << "ERROR: uncaught_exception still set." << std::endl;
           else
-            std::cout << "OK";
+            std::cout << "OK" << std::endl;
```

Comment:
The addition of `std::endl`s here looks also odd to me? Was there something that did not work here without these?

Reply:
See above